### ISSUE TITLE
1.18.2 Placeholder parsing error

### ISSUE THREAD

**Issue by YrChen1001** [kangarko/ChatControl#1970]
```
[17:42:54 INFO]: [ChatControlRed] !-----------------------------------------------------!
[17:42:54 INFO]: [ChatControlRed]  IMPORTANT: PREVENTED SERVER CRASH FROM PLACEHOLDERAPI
[17:42:54 INFO]: [ChatControlRed]  Replacing a variable using PlaceholderAPI took
[17:42:54 INFO]: [ChatControlRed]  longer than our maximum limit (1 second) and
[17:42:54 INFO]: [ChatControlRed]  was forcefully interrupted to prevent your
[17:42:54 INFO]: [ChatControlRed]  server from crashing. This is not error on
[17:42:54 INFO]: [ChatControlRed]  our end, please contact the expansion author.
[17:42:54 INFO]:
[17:42:54 INFO]: [ChatControlRed]  Variable: vault
[17:42:54 INFO]: [ChatControlRed]  Player: null
[17:42:54 INFO]: [ChatControlRed] !-----------------------------------------------------!
[17:42:54 WARN]: [ChatControlRed] Task #230477 for ChatControlRed v10.12.1 generated an exception
java.lang.ThreadDeath: null
        at java.lang.Thread.stop(Thread.java:942) ~[?:?]
        at org.mineacademy.chatcontrol.lib.model.PlaceholderAPIHook.lambda$setPlaceholders$0(HookManager.java:2231) ~[ChatControl-Red.jar:?]
        at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftTask.run(CraftTask.java:101) ~[paper-1.18.2.jar:git-Paper-386]
        at org.bukkit.craftbukkit.v1_18_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:483) ~[paper-1.18.2.jar:git-Paper-386]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1532) ~[paper-1.18.2.jar:git-Paper-386]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:483) ~[paper-1.18.2.jar:git-Paper-386]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1456) ~[paper-1.18.2.jar:git-Paper-386]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1226) ~[paper-1.18.2.jar:git-Paper-386]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:316) ~[paper-1.18.2.jar:git-Paper-386]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

Minecraft Version: 1.18.2
MySQL: true
BungeeCord: true

**Comment by kangarko** [comment]
The problem is in the "vault" variable, not really sure what variable is that but I assume your permissions or other plugin formatting such variable has a huge database (such as on your disk) that is causing this hang up. You need to dig deeper and trim your database size or contact your expansion developer.

**Comment by YrChen1001** [comment]
I found this problem occurs when the player never joins the server and then sends a message to Minecraft from Discord

**Comment by kangarko** [comment]
Thanks for telling me, going to review in 24h

**Comment by kangarko** [comment]
Sorry, after my investigation I found that your vault variable sends a blocking request to find the offline player's name or UUID. This needs to be fixed by your expansion author and not us, unfortunately sending an Internet request to Mojang servers to get that info will always take a while so I think they can't fix it. 

So what I did is I extended the waiting period from 1 second to 4 seconds, should be enough.
